### PR TITLE
chore(docs): bump glob version used in docs 

### PR DESCRIPTION
**Pull request by esauerbo**
#### Description of changes

Added glob as a dependency in packages where it was being used to keep it at latest version 
`docs/src/utils/getContentPaths.ts`
`packages/ui/scripts/generateCSS.ts`

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "csstype": "^3.1.1",
+    "glob": "^10.3.10",
```
Comment (reesscot) [inline packages/ui/package.json:48]: I think it's a dev dependency in ui package, not a main dep

File: docs/package.json [hunk 2]
```diff
@@ -65,6 +65,7 @@
     "canvas": "^2.11.2",
     "dotenv-safe": "^8.2.0",
     "eslint-config-next": "^13.5.5",
+    "glob": "^10.3.10",
```
Comment (reesscot) [inline docs/package.json:68]: How is it used in docs? I think this is only used during build. I think it's a dev dependency here too

**Review by reesscot — COMMENTED** [review]
I think it's a devDependency for both of these. Also need to run yarn install to update yarn lock file

File: docs/package.json (hunk 2 above)
Comment (esauerbo) [reply docs/package.json:68]: Yeah it's just used in the one file `docs/src/utils/getContentPaths.ts`, did you mean if it's only used in build we don't need to declare it here?

File: docs/__tests__/__snapshots__/sitemap.test.ts.snap [hunk 3]
```diff
@@ -1,166 +1,43 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Sitemap Snapshot 1`] = `
-"/, 
-/android,
```
Comment (reesscot) [inline docs/__tests__/__snapshots__/sitemap.test.ts.snap:5]: looks like upgrading glob broke our sitemap script?

Comment (esauerbo) [reply docs/__tests__/__snapshots__/sitemap.test.ts.snap:5]: Actually I don't think this is related to this change specifically, not sure what caused it but I noticed the sitemap snapshot was failing in a few other docs prs

File: docs/src/utils/getContentPaths.ts [hunk 4]
```diff
@@ -10,8 +10,7 @@ export type GetContentPaths = (pattern?: string) => Promise<string[]>;
 export const getContentPaths: GetContentPaths = async (
```
Comment (reesscot) [inline docs/src/utils/getContentPaths.ts:10]: Is it still an async function if we use globSync?

**Review by reesscot — APPROVED** [review]
LGTM!

**Review by ioanabrooks — APPROVED** [review]

**Pull request merged**